.golangci.yml

```diff
@@ -146,6 +146,14 @@ linters-settings:
         files:
           - "!**/jptrace/**"
 
+      # TODO: remove once we have upgraded to Go 1.23
+      disallow-iter:
+        deny:
+         - pkg: iter
```

Comment:
this is very odd - our go.mod says
```
go 1.22.7
```
so it should not have compiled w/ the std `iter` package